geode-core/src/distributedTest/java/org/apache/geode/internal/cache/DeltaForceSizingFlagDUnitTest.java

```diff
@@ -0,0 +1,460 @@
+          new TestObjectSizer(), EvictionAction.OVERFLOW_TO_DISK));
+
+      DiskStoreFactory diskStoreFactory = cache.createDiskStoreFactory();
+      diskStoreFactory.setDiskDirs(getMyDiskDirs());
+      diskStoreFactory.create(RR_DISK_STORE_NAME);
+      regionFactory.setDiskStoreName(RR_DISK_STORE_NAME);
+
+      regionFactory.create(TEST_REGION_NAME);
+    });
+  }
+
+  private void assertValueType(VM vm, final ValueType expectedType) {
+    vm.invoke("assertValueType", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      assertNotNull(cache);
+      LocalRegion region = (LocalRegion) cache.getRegion(TEST_REGION_NAME);
+      Object value = region.getValueInVM(DeltaForceSizingFlagDUnitTest.DELTA_KEY);
+      switch (expectedType) {
+        case RAW_VALUE:
+          assertFalse("Value was " + value + " type " + value.getClass(),
+              (value instanceof CachedDeserializable));
+          break;
+        case CD_SERIALIZED:
+          assertTrue("Value was " + value + " type " + value.getClass(),
+              value instanceof CachedDeserializable);
+          assertTrue("Value not serialized",
+              ((CachedDeserializable) value).getValue() instanceof byte[]);
+          break;
+        case CD_DESERIALIZED:
+          assertTrue("Value was " + value + " type " + value.getClass(),
+              value instanceof CachedDeserializable);
+          assertFalse("Value was serialized",
+              (((CachedDeserializable) value).getValue() instanceof byte[]));
+          break;
+        case EVICTED:
+          assertNull(value);
+          break;
+      }
+    });
+  }
+
+  private static File[] getMyDiskDirs() {
+    long random = new Random().nextLong();
+    File file = new File(Long.toString(random));
+    assertTrue(file.mkdirs());
+    return new File[] {file};
+  }
+
+  private void createPR(MemberVM memberVM) {
+    memberVM.invoke("Create partitioned region", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      assertNotNull(cache);
+
+      RegionFactory<Integer, TestDelta> regionFactory = cache.createRegionFactory();
+
+      regionFactory.setDiskSynchronous(true);
+      PartitionAttributesFactory<Integer, TestDelta> paf =
+          new PartitionAttributesFactory<>();
+      paf.setRedundantCopies(1);
+      PartitionAttributes<Integer, TestDelta> prAttr = paf.create();
+      regionFactory.setPartitionAttributes(prAttr);
+      regionFactory.setDataPolicy(DataPolicy.PARTITION);
+      regionFactory.setSubscriptionAttributes(new SubscriptionAttributes(InterestPolicy.ALL));
+      regionFactory.create(TEST_REGION_NAME);
+    });
+  }
+
+  private static class TestObjectSizer implements ObjectSizer {
+    private final AtomicInteger invocations = new AtomicInteger();
+
+    @Override
+    public int sizeof(Object o) {
+      logger.debug("TestObjectSizer invoked"/* , new Exception("stack trace") */);
```

Comment:
The commented out code here should be removed, possibly along with the rest of the log message. It doesn't seem to add much in terms of helping a developer debug a possible future failure. If the logging is left in, it should be changed to info level.